varcode/variant.py

```diff
@@ -101,8 +109,8 @@ def __init__(self,
         # normalize the variant by trimming any shared prefix or suffix
         # between ref and alt nucleotide sequences and then
         # offset the variant position in a strand-dependent manner
-        trimmed_ref, trimmed_alt, prefix, suffix = trim_shared_flanking_strings(
-            self.original_ref, self.original_alt)
+        (trimmed_ref, trimmed_alt, prefix, suffix) = (
+            trim_shared_flanking_strings(self.original_ref, self.original_alt))
```

Comment:
Nitpick, but I don't think you need the extra parens around the function call, PEP8 seems pretty happy with arguments on the next line: http://legacy.python.org/dev/peps/pep-0008/#indentation